### PR TITLE
Remove usage of Condense (TxId (GenTx blk)) instances

### PR DESCRIPTION
**Pull request by intricate**

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/ConvertTxId.hs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+-- | Convert a transaction ID to raw bytes.
+class ConvertTxId blk where
+  txIdToRawBytes :: TxId (GenTx blk) -> ByteString
+
+instance ConvertTxId ByronBlock where
+  txIdToRawBytes (ByronTxId txId) = Byron.Crypto.abstractHashToBytes txId
+  txIdToRawBytes (ByronDlgId dlgId) = Byron.Crypto.abstractHashToBytes dlgId
+  txIdToRawBytes (ByronUpdateProposalId upId) =
+    Byron.Crypto.abstractHashToBytes upId
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/ConvertTxId.hs:36]: Is your editor auto-wrapping after `80` columns?  The stylish haskell rules allow up to `100`

File: cardano-node/src/Cardano/Tracing/ConvertTxId.hs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import           Ouroboros.Consensus.Ledger.SupportsMempool (GenTx)
+import           Ouroboros.Consensus.Mock.Ledger.Block (SimpleBlock, TxId (..))
+import           Ouroboros.Consensus.Shelley.Ledger.Block (ShelleyBlock)
+import           Ouroboros.Consensus.Shelley.Ledger.Mempool (TxId (..))
+import           Ouroboros.Consensus.TypeFamilyWrappers
+import qualified Shelley.Spec.Ledger.TxData as Shelley
+
+-- | Convert a transaction ID to raw bytes.
+class ConvertTxId blk where
+  txIdToRawBytes :: TxId (GenTx blk) -> ByteString
```
Comment (newhoggy) [inline cardano-node/src/Cardano/Tracing/ConvertTxId.hs:30]: Should we have tests for these?

File: cardano-node/src/Cardano/Tracing/ConvertTxId.hs (hunk 1 above)
Comment (intricate) [reply cardano-node/src/Cardano/Tracing/ConvertTxId.hs:36]: I just stick with 80 personally, but I can keep it to 100.

File: cardano-cli/src/Cardano/CLI/Byron/Query.hs [hunk 3]
```diff
@@ -57,13 +56,13 @@ runGetLocalNodeTip networkId = do
       tip <- getLocalTip connctInfo
       putTextLn (getTipOutput tip)
   where
-    -- TODO: Remove `Condense (HeaderHash blk)`.
-    getTipOutput :: forall blk. Condense (HeaderHash blk) => Tip blk -> Text
```
Comment (Jimbo4350) [inline cardano-cli/src/Cardano/CLI/Byron/Query.hs:61]: :tada:

File: cardano-cli/src/Cardano/CLI/Byron/Query.hs [hunk 4]
```diff
@@ -2,30 +2,29 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE RankNTypes #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
```
Comment (Jimbo4350) [inline cardano-cli/src/Cardano/CLI/Byron/Query.hs:5]: Is `TypeApplications` necessary here?

Comment (intricate) [reply cardano-cli/src/Cardano/CLI/Byron/Query.hs:5]: Yeah

File: cardano-node/src/Cardano/Tracing/ConvertTxId.hs (hunk 2 above)
Comment (Jimbo4350) [reply cardano-node/src/Cardano/Tracing/ConvertTxId.hs:30]: Maybe a golden test?

**Pull request merged**